### PR TITLE
Add security headers

### PR DESCRIPTION
**Pull request by jonashendrickx**

### CODE REVIEW

File: src/AdminConsole/Program.cs [hunk 1]
```diff
@@ -113,6 +113,7 @@ void RunTheApp()
     services.ConfigureApplicationCookie(o =>
     {
         o.Cookie.Name = "AdminConsoleSignIn";
+        o.Cookie.SameSite = SameSiteMode.Strict;
```
Comment (abergs) [inline src/AdminConsole/Program.cs:116]: I think this is good to harden the security. We already do CSRF protection using the antiforgery-token, but having multiple levels of protection is good.

Have you tested the behaviour of the application when clicking links to the admin console? There are many places where we link or could be linking to our app (Email with magic link, from documentation, notification emails).

Are those links honoured? Or will the user be sent to a unauthorised-page, and then redirect to the index page?

CleanShot 2024-02-02 at 13 01 36

Comment (jonashendrickx) [reply src/AdminConsole/Program.cs:116]: @jrmccannon or @Tyrrrz Do either of you guys want to help me test this on Monday with magic links? I can test the other scenarios.

Comment (jrmccannon) [reply src/AdminConsole/Program.cs:116]: Running locally, magic links worked fine for logging into AdminConsole.

**Review by jrmccannon — APPROVED** [review]

File: src/AdminConsole/Program.cs (hunk 1 above)
Comment (abergs) [reply src/AdminConsole/Program.cs:116]: @jrmccannon You tested a magic link from a file open in vscode/ide, not from another website, like gmail.com -- right?

**Pull request merged**